doc/content/manuals/theory/index.md

```diff
@@ -148,17 +156,17 @@ To model the mechanical behavior of a material, three components are need to be
 Details about stress calculation for two different constitutive models are presented below.
 
```

Comment:
In the strain section above I have added a link to the moose tensor mechanics documentation from a PR that was open during the SQA meeting. Can this link be replaced with a better link that shows the current version of the Tensor mechanics documentation?